services/wallet/thirdparty/rarible/client.go

```diff
@@ -390,6 +390,17 @@ func (o *Client) FetchAssetsByCollectibleUniqueID(ctx context.Context, uniqueIDs
 	return ret, nil
 }
 
+func (o *Client) FetchCollectibleSocialsByUniqueID(ctx context.Context, uniqueID thirdparty.CollectibleUniqueID) (thirdparty.CollectionSocials, error) {
+	resp, err := o.FetchCollectionsDataByContractID(ctx, []thirdparty.ContractID{uniqueID.ContractID})
```

Comment:
Rarible's socials data is incomplete, just return `ErrEndpointNotSupported` here so other providers are used instead.